### PR TITLE
chore: 6.2.0 release article, documentation

### PR DESCRIPTION
**Pull request by bsmth**
This release contains the following changes:

__TODO:__
- [x]  Add a blog post about version 6.2.0
- [x] document new server configs for JIT
- [x] latest by syntax (https://github.com/questdb/questdb/pull/1739)
- [ ] merge in light theme from master (depends on https://github.com/questdb/questdb.io/pull/669)
- [ ] demo box requires 6.2.0 deployed

### CODE REVIEW

File: docs/concept/jit-compiler.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+---
+
+QuestDB includes a JIT compiler which is run on queries (and sub-queries) that
+perform a full scan over a table or table partitions. The main goal behind this
+feature is to improve performance for filters with arithmetical expressions. To
+do so, the JIT compiler emits machine code with a single function that may also
+use SIMD (vector) instructions.
+
+For details on the implementation, motivation, and internals of this feature
+compiler, see our
```
Comment (puzpuzpuz) [inline docs/concept/jit-compiler.md:17]: nit: `feature compiler` should probably be just `feature` here

File: docs/concept/jit-compiler.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+The JIT compiler is currently a beta feature, and therefore performance may vary
+as enhancements and improvements are added. For not-yet-implemented
+functionality, see the [known limitations](#known-limitations) section below.
+
+:::
+
+## Queries eligible for JIT compilation
+
+The types of queries that are eligible for performance improvements via JIT
+compilation are those which contain `WHERE` clauses. Here's some examples which
```
Comment (puzpuzpuz) [inline docs/concept/jit-compiler.md:33]:
```suggestion
compilation are those which contain `WHERE` clauses. Here are some examples which
```

**Review by puzpuzpuz — APPROVED** [review]

File: docs/concept/jit-compiler.md (hunk 1 above)
Comment (bsmth) [reply docs/concept/jit-compiler.md]: fixed in https://github.com/questdb/questdb.io/pull/674/commits/f05e1d7e5f3f4ff531a3e41eb98f3e4e9650ff88

File: docs/concept/jit-compiler.md (hunk 2 above)
Comment (bsmth) [reply docs/concept/jit-compiler.md]: fixed in https://github.com/questdb/questdb.io/pull/674/commits/f05e1d7e5f3f4ff531a3e41eb98f3e4e9650ff88

**Pull request merged**